tests/e2e/e2e_common.go

```diff
@@ -148,7 +148,9 @@ const (
 	vsphereTKGSystemNamespace                 = "vmware-system-tkg"
 	waitTimeForCNSNodeVMAttachmentReconciler  = 30 * time.Second
 	wcpServiceName                            = "wcp"
+	vmcWcpHost                                = "10.2.224.24"
```

Comment:
drop a comment here mentioning why we are hardcoding to this IP
better to have this parameterised but set to this IP by default

Reply:
sure, I'll add a comment. For on-prem to get the host Im planning to add a separate method like this. so we can pass this host for further tests
`wcpHost := getWCPHost(wcpCluster, e2eVSphere.Config.Global.VCenterHostname, sessionID)`